aries/aries_vcx_core/src/wallet/askar/mod.rs

```diff
@@ -0,0 +1,191 @@
+use aries_askar::{
+    entry::EntryTag,
+    kms::{KeyAlg, KeyEntry, LocalKey},
+    PassKey, Session, Store, StoreKeyMethod,
+};
+
+use self::{askar_utils::local_key_to_bs58_name, rng_method::RngMethod};
+use super::{
+    base_wallet::{did_data::DidData, BaseWallet},
+    constants::DID_CATEGORY,
+    utils::key_from_base58,
+};
+use crate::errors::error::{AriesVcxCoreError, AriesVcxCoreErrorKind, VcxCoreResult};
+
+mod askar_did_wallet;
+mod askar_record_wallet;
+mod askar_tags;
+mod askar_utils;
+mod crypto_box;
+mod entry;
+mod packing;
+mod packing_types;
+mod rng_method;
+mod sig_type;
+
+#[derive(Debug)]
+pub struct AskarWallet {
+    backend: Store,
+    profile: Option<String>,
+}
+
+impl BaseWallet for AskarWallet {}
+
+impl AskarWallet {
+    pub async fn create(
+        db_url: &str,
+        key_method: StoreKeyMethod,
+        pass_key: PassKey<'_>,
+        recreate: bool,
+        profile: Option<String>,
```

Comment:
i was looking at askar code trying to figure out what `profile` is ... what it is? Seems like some wallet-scoped multi-tenancy identifier?

Reply:
Yeah, you can essentially have multiple "accounts" in a single wallet:

https://github.com/hyperledger/aries-askar/blob/main/docs/storage.md#profiles